frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-topology/__tests__/WhenExpressionDecorator.spec.tsx

```diff
@@ -14,20 +21,79 @@ describe('WhenExpressionDecorator', () => {
   const props: WhenExpressionDecoratorProps = {
     width: 10,
     height: 10,
-    color: 'white',
+    status: 'Failed',
```

Comment:
Maybe use the enum instead? Not sure it matters as I doubt it will ever change, so call this a nit.